src/commands/serve.js

```diff
@@ -241,7 +241,7 @@ async function serve(command) {
 
   await executeBeforeBuildPlugins(programInfo, { createContent });
 
-  programInfo.port = command.port || 5000;
+  programInfo.port = command.port;
```

Comment:
I have to remove conditional from here as I wanted to know inside the createServer function if the port was user defined or default.

Also, we have a default in two places which was weird.